proteuslib/tests/test_chlorination.py

```diff
@@ -497,77 +509,29 @@
                    "temperature": pyunits.K},
     "equilibrium_reactions": {
         "NH2Cl_K": {

```

Comment:
Should this be "NCl3_K" since only the overall reaction is included now?

Reply:
Done